node/src/effect.rs

```diff
@@ -943,6 +943,27 @@ impl<REv> EffectBuilder<REv> {
             .await
     }
 
+    /// An equivocation has been detected.
+    pub(crate) async fn announce_equivocation_event(
```

Comment:
I think it should be renamed now to `announce_fault_event`. This method can be called from anywhere which means it could be called by other implementations of the consensus protocol which do not have the concept of equivocations.